### PR TITLE
[javasound] Readme

### PR DESCRIPTION
**Pull request by davidgraeff**
Initiative to provide a Readme to all addon directories. Sometimes you really don't know what they are for.

Signed-off-by: davidgraeff <david.graeff@web.de>

### CODE REVIEW

File: addons/io/org.openhab.io.javasound/README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Default audio sink and source
+
+This IO bundle provides a Java API based `AudioSink` and `AudioSource`
+to let a user play and capture audio on the host system.
```
Comment (Confectrician) [inline addons/io/org.openhab.io.javasound/README.md]: Second line needed here?

Comment (davidgraeff) [reply addons/io/org.openhab.io.javasound/README.md]: Not really. It doesn't hurt though ^^

Comment (Confectrician) [reply addons/io/org.openhab.io.javasound/README.md]: We usually say one line per sentence.
In my view it is inconsistent to tell people splitting their texts up and then use multiple lines for one sentence on the other hand.

The generated view will be the same anyway, so its just my opinion.

Comment (davidgraeff) [reply addons/io/org.openhab.io.javasound/README.md]: I'll quickly change it. Will just not be signed off then anymore.

**Pull request merged**